Content.Server/Clothing/Components/ClothingComponent.cs

```diff
@@ -33,12 +33,18 @@ public class ClothingComponent : ItemComponent, IUse
         [DataField("HeatResistance")]
         private int _heatResistance = 323;
 
+        [DataField("Sterile")]
```

Comment:
datafields should be `camelCase`, the rest of the component doesn't really follow this but uhhhhh

Reply:
also this should really be its own component, not just slapped onto clothing?

Reply:
you already made another component for this so just put it in there